source/features/link-to-gihub-io.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import LinkIcon from 'octicon/link.svg';
+import elementReady from 'element-ready';
+import * as pageDetect from 'github-url-detection';
+import LinkExternalIcon from 'octicon/link-external.svg';
+
+import features from '.';
+import {getRepositoryInfo} from '../github-helpers';
+
+function getGithubIoLink(): string {
+	return `${getRepositoryInfo().owner!}.github.io`;
+}
+
+async function repoTabInit(): Promise<void> {
+	const repos = select.all<HTMLAnchorElement>('a[href$=".github.io"][itemprop="name codeRepository"]');
+	for (const repo of repos) {
+		const path = new URL(repo.href).pathname.slice(1);
+		const repoInfo = getRepositoryInfo(path);
+
+		repo.parentElement!.append(
+			<a
+				href={`https://${repoInfo.name!}`}
+				target="_blank"
+				rel="noopener noreferrer"
+			>
+				<LinkExternalIcon/>
+			</a>
+		);
+	}
+}
+
+async function githubIoRepoInit(): Promise<void | false> {
+	const githubIoLink = getGithubIoLink();
+
+	if (select.exists('.octicon-link', await elementReady('.BorderGrid-cell'))) {
+		return false;
+	}
+
+	select('.BorderGrid-cell')!.append(
+		<div className="mt-3 d-flex flex-items-center">
+			<LinkIcon className="mr-2"/>
+			<span className="flex-auto min-width-0 css-truncate css-truncate-target width-fit">
+				<a
+					role="link"
+					target="_blank"
+					className="text-bold"
+					rel="noopener noreferrer"
+					href={`https://${githubIoLink}`}
```

Comment:
As before, the URL is easily available and we don't need to call` getGithubIoLink`
```suggestion
					href={`https://${getRepositoryInfo().name}`}
```

Reply:
`githubIoLink` is also needed in the line below, to mark the text of the anchor tag, which is why I'd declared it as a variable above. Is that something that we can do away with and call the function twice.

What I can do, is remove `getGithubIoLink` entirely and replace it with
```
const githubIoLink = getRepositoryInfo().name!
```

Reply:
Yeah that works

Reply:
Done!